operator/v1/types_console.go

```diff
@@ -21,13 +21,27 @@ type ConsoleSpec struct {
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
 	// +optional
-	Customization ConsoleCustomization `json:"customization"`
+	Customization ConsoleCustomization `json:"customization,omitempty"`
+	// providers contains configuration for using specific service providers.
+	Providers ConsoleProviders `json:"providers,omitempty"`
```

Comment:
Ditto wrt `omitempty`.

Reply:
`ProviderConfig` ? The `Providers` implicate list or array?

Reply:
As @benjaminapetersen mentioned, we will handle each provider separately

Reply:
So maybe we should start naming providers, similarly how we do it in oauth?

Reply:
Nvmd, it's following the current patterns. I misread it .

Reply:
You got it! That's exactly the pattern we were aiming to follow.